cfgen/cfgen

```diff
@@ -1431,51 +1500,54 @@ def pool_drives(m0conf: Dict[Oid, Any],
     if ptype is PoolT.dix:
         cas = [(svc_id, m0conf[svc_id].ctrl_id)
                for svc_id in cluster_svcs(m0conf, SvcT.M0_CST_CAS)]
+        path = '/dev/null'
+        nodes = cluster_desc.get('nodes')
+        if nodes:
+            for nodes in nodes:
+                m0servers = nodes.get('m0_servers')
+                if m0servers:
```

Comment:
Same here, no additional if loop required.

Reply:
If I remove this `if` condition here, then it throws error like:
`--> Checking cfgen
cfgen:1505: error: Item "None" of "Optional[Any]" has no attribute "__iter__" (not iterable)
`